flavors/midnight-rose-pine.theme.css

```diff
@@ -0,0 +1,95 @@
+/**
+ * @name Rose Pine
+ * @description A dark, rounded discord theme.
+ * @author refact0r, anubis, PsychotherapistSam
+ * @version 1.6.2
+ * @invite nz87hXyvcy
+ * @website https://github.com/refact0r/midnight-discord
+ * @source https://github.com/refact0r/midnight-discord/blob/master/flavors/midnight-rose-pine.theme.css
+ * @authorId 508863359777505290
+ * @authorLink https://www.refact0r.dev
+*/
+
+/* IMPORTANT: make sure to enable dark mode in discord settings for the theme to apply properly!!! */
+
+@import url('https://refact0r.github.io/midnight-discord/midnight.css');
+
+/* customize things here */
+:root {
+    /* font, change to 'gg sans' for default discord font */
+    /*--font: 'figtree';*/
+    --font: 'Chillax';
+
+    /* top left corner text */
+    --corner-text: 'Rose Pine';
+
+    /* color of status indicators and window controls */
+    --online-indicator: hsl(189deg, 43%, 73%); /* change to #23a55a for default green */
+    --dnd-indicator: hsl(343deg, 76%, 68%); /* change to #f13f43 for default red */
+    --idle-indicator: hsl(35deg, 88%, 72%); /* change to #f0b232 for default yellow */
+    --streaming-indicator: hsl(267deg, 57%, 78%); /* change to #593695 for default purple */
+
+    /* accent colors */
+    --accent-1: hsl(2deg, 55%, 83%); /* links */
+    --accent-2: hsl(2deg, 55%, 83%); /* general unread/mention elements */
+    --accent-3: hsl(2deg, 55%, 83%); /* accent buttons */
+    --accent-4: hsl(2deg, 55%, 83%); /* accent buttons when hovered */
+    --accent-5: hsl(2deg, 55%, 83%); /* accent buttons when clicked */
+    --mention: hsl(247deg, 23%, 15%); /* mentions & mention messages */
+    --mention-hover: hsl(248deg, 25%, 18%); /* mentions & mention messages when hovered */
+
+    /* text colors */
+    --text-0: hsl(249deg, 22%, 12%); /* text on colored elements */
+    --text-1: hsl(248deg, 15%, 61%); /* other normally white text */
+    --text-2: hsl(245deg, 50%, 91%); /* headings and important text */
+    --text-3: hsl(245deg, 50%, 91%); /* normal text */
+    --text-4: hsl(248deg, 15%, 61%); /* icon buttons and channels */
+    --text-5: hsl(248deg, 15%, 61%); /* muted channels/chats and timestamps */
+
+    /* background and dark colors */
+    --bg-1: hsl(248deg, 25%, 18%); /* dark buttons when clicked */
+    --bg-2: hsl(249deg, 12%, 47%); /* dark buttons */
+    --bg-3: hsl(247deg, 23%, 15%); /* spacing, secondary elements */
+    --bg-4: hsl(249deg, 22%, 12%); /* main background color */
+    --hover: hsl(247deg, 23%, 15%); /* buttons when hovered */
+    --active: hsl(248deg, 25%, 18%); /* channels and buttons when clicked or selected */
+    --message-hover: hsl(247deg, 23%, 15%); /* messages when hovered */
```

Comment:
Although this might need to be adjusted a bit because when messages with invite links are hovered the invite link background is the same color as the hovered message, wasn't sure

Reply:
i've changed the message hover to go darker, preventing this from happening.